.github/workflows/issue_comment.yml

```diff
@@ -19,17 +19,27 @@ jobs:
         shell: bash
         run: echo $COMMENT | sed "s/\\\n/. /g; s/\\\r//g; s/[^a-zA-Z0-9 &().,:]//g" | xargs -I {} curl -s POST "$WEBHOOK_URL" -H "Content-Type:application/json" --data '{"comment":"{}", "commentUrl":"'$COMMENT_URL'", "user":"'$USER'"}'
 
-  remove-pending-response-label:
+  adjust-labels:
     runs-on: ubuntu-latest
     permissions:
       issues: write
-    if: ${{ !github.event.issue.pull_request && contains(github.event.issue.labels.*.name, 'pending-response') }}
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      ISSUE_NUMBER: ${{ github.event.issue.number }}
+      REPOSITORY_NAME: ${{ github.event.repository.full_name }}
     steps:
-      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 #v4.1.1
-      - name: remove unnecessary labels after closing
+      - name: remove pending-community-response when new comment received
+        if: ${{ !contains(fromJSON('["MEMBER", "OWNER"]'), github.event.comment.author_association) && !github.event.issue.pull_request }}
+        shell: bash
+        run: |
+          gh issue edit $ISSUE_NUMBER --repo $REPOSITORY_NAME --remove-label "pending-community-response"
+      - name: add pending-maintainer-response when new community comment received
+        if: ${{ !contains(fromJSON('["MEMBER", "OWNER"]'), github.event.comment.author_association) }}
```

Comment:
IIRC for this filter to work you will need to set your [aws-amplify org membership visibility to **Public**](https://github.com/orgs/aws-amplify/people), otherwise the runner won't be able to see it.

Reply:
This is no different than the filter above that was already being used?

Reply:
Yeah, everybody in the Swift team made their membership public in order for this to work. You guys will probably need to do the same.

Reply:
Got it, I don't remember doing this but we can check. We've had this type of filter used in other places and working well, so maybe we had already done that as well.

Reply:
Cool! I might be mixing things up with the ops metrics lambda needing this permission instead of the GH action 😅 
But I do remember it was needed _somewhere_ 😬